test/plugins/features/movies/controller.test.js

```diff
@@ -25,12 +25,26 @@ describe('movie controller', () => {
 
   describe('get', () => {
 
-    it('gets movies prior to a given release_year', () => {
-      const payload = { release_year: 1984 };
+    it('gets movies prior that match search params', () => {
```

Comment:
Instead of having one test that tests everything (e.g. testing `year_start`, `year_end`, and `title`), it's better to opt for individual tests for each thing. If you had that, you'd have a test that would fail when you tried to filter by title since the interpolation had a bug in it.